src/api/attack/util.rs

```diff
@@ -190,41 +192,50 @@ pub fn add_game(
 }
 
 pub fn fetch_attack_history(
-    attacker_id: i32,
     user_id: i32,
+    page: i64,
+    limit: i64,
     conn: &mut PgConnection,
-) -> Result<GameHistoryResponse> {
+) -> Result<HistoryboardResponse> {
     use crate::schema::{game, levels_fixture, map_layout};
-    let attacker = fetch_user(conn, attacker_id)?.ok_or(AuthError::UserNotFound)?;
     let joined_table = game::table
         .inner_join(map_layout::table.inner_join(levels_fixture::table))
         .inner_join(user::table.on(game::defend_id.eq(user::id)));
```

Comment:
Remove unnecessary joins if not needed

Reply:
We can reach the levels_fixture table, which is used while checking for replay availability, from game table through map_layout table